ttnn/cpp/ttnn/operations/ccl/reduce_scatter/device/kernels/worker_interleaved_ring_reduce_scatter_sender.cpp

```diff
@@ -260,7 +262,8 @@ void kernel_main() {
         pop_filler_pages_from_cb(cb_id_in0, 1);
     }
 
-    if (num_transfers > 0) {
+    if (sends_to_edm) {
         sender.close();
     }
+    WAYPOINT("DONE");
```

Comment:
Remove WAYPOINT for release code

Reply:
Is this a coding standard somewhere? It's in iur datfaflow API code and elsewhere and is not compiled in by default

Reply:
Oh in that case it's fine, I typically try removing anything debug from what is sent out but you are right it is essentially a comment with the right flags